.github/workflows/docker-build-static.yml

```diff
@@ -523,6 +562,39 @@ jobs:
             suffix=-${{ matrix.container }}
           tags: ${{ matrix.tags }}
 
+      - name: Docker ${{ matrix.container }}-rootless meta
+        id: meta-rootless
+        uses: docker/metadata-action@v5
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            nitnelave/lldap
+            lldap/lldap
+            ghcr.io/lldap/lldap
+          # Wanted Docker tags
+          # vX-alpine
+          # vX.Y-alpine
+          # vX.Y.Z-alpine
+          # latest
+          # latest-alpine
+          # stable
+          # stable-alpine
+          # YYYY-MM-DD
+          # YYYY-MM-DD-alpine
+          #################
+          # vX-debian
+          # vX.Y-debian
+          # vX.Y.Z-debian
+          # latest-debian
+          # stable-debian
+          # YYYY-MM-DD-debian
+          #################
+          # Check matrix for tag list definition
+          flavor: |
+            latest=false
```

Comment:
Latest should be false?

Reply:
Yes, the latest thing taken care on matrix, this part.
```
444     strategy:
445       matrix:
446         container: ["debian","alpine"]
447         include:
448           - container: alpine
449             platforms: linux/amd64,linux/arm64,linux/arm/v7
450             tags: |
451                   type=ref,event=pr
452                   type=semver,pattern=v{{version}}
453                   type=semver,pattern=v{{major}}
454                   type=semver,pattern=v{{major}}.{{minor}}
455                   type=semver,pattern=v{{version}},suffix=
456                   type=semver,pattern=v{{major}},suffix=
457                   type=semver,pattern=v{{major}}.{{minor}},suffix=
458                   type=raw,value=latest,enable={{ is_default_branch }}
459                   type=raw,value=stable,enable=${{ startsWith(github.ref, 'refs/tags/v') }}
460                   type=raw,value=stable,enable=${{ startsWith(github.ref, 'refs/tags/v') }},suffix=
461                   type=raw,value=latest,enable={{ is_default_branch }},suffix=
462                   type=raw,value={{ date 'YYYY-MM-DD' }},enable={{ is_default_branch }}
463                   type=raw,value={{ date 'YYYY-MM-DD' }},enable={{ is_default_branch }},suffix=
464           - container: debian
465             platforms: linux/amd64,linux/arm64,linux/arm/v7
466             tags: |
467                   type=ref,event=pr
468                   type=semver,pattern=v{{version}}
469                   type=semver,pattern=v{{major}}
470                   type=semver,pattern=v{{major}}.{{minor}}
471                   type=raw,value=latest,enable={{ is_default_branch }}
472                   type=raw,value=stable,enable=${{ startsWith(github.ref, 'refs/tags/v') }}
473                   type=raw,value={{ date 'YYYY-MM-DD' }},enable={{ is_default_branch }}
```